test/gateway/eip20_cogateway/confirm_revert_stake_intent.js

```diff
@@ -0,0 +1,308 @@
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+const EIP20CoGateway = artifacts.require("TestEIP20CoGateway");
+const UtilityToken = artifacts.require("MockUtilityToken");
+const BN = require('bn.js');
+const Utils = require("./../../test_lib/utils");
+const coGatewayUtils = require('./helpers/co_gateway_utils.js');
+const EventDecoder = require('../../test_lib/event_decoder.js');
+const messageBus = require('../../test_lib/message_bus.js');
+const StubData = require('../../data/stake_revoked_1.json');
+
+let MessageStatusEnum = messageBus.MessageStatusEnum;
+
+contract('EIP20CoGateway.confirmRevertStakeIntent() ', function (accounts) {
+  let utilityToken,
+    coGateway,
+    owner = accounts[0], messageHash, amount,
+    stakeRequest = StubData.gateway.stake.params;
+
+  beforeEach(async function () {
+
+    let organization = accounts[2];
+
+    utilityToken = await UtilityToken.new(
+      accounts[9],
+      "",
+      "",
+      18,
```

Comment:
Named variables would help understanding (instead of magic values).

Reply:
Added comment